proposed/phpdoc-tags.md

```diff
@@ -955,7 +933,7 @@ Another example is to document the variable in a foreach explicitly; many IDEs
 use this information to help you with auto-completion:
 
 ```php
-/** @var \Sqlite3 $sqlite */
+/* @var \Sqlite3 $sqlite */
```

Comment:
I just checked with phpstorm, `/*` and `/**` do work both. But `/**` has some improved auto-completion. Where `/*` requires some more typing. 

I don't know if this document states anything about inline comments like

use `//` for single line inline comments 
use `/*` for multi line inline comments
use `/**` for docblocks, above `fqsen`

at this moment `//` doesn't support tags, at least in phpstorm it doesn't. And I have never seen anythink like that in the wild. But this change triggers me that there should be some statement about inline docs.

Reply:
I went to the `phpdoc.md` format PSR to look at having these three syntaxes mentioned.  I found original verbiage that says the `/** @var */` syntax for `foreach` variables is explicitly **not** defined in the standard.  So, this begs the question of whether or not we *do* want it defined in the standard.  If yes, then the verbiage I mention gets removed... if no, then this `@var` example above gets removed.

Reply:
My intuition is that everyone *does* want the `foreach @var` usage to be included in the var tag definition, so I've removed the "not in standard" verbiage.